src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -546,6 +552,11 @@ export default {
 		loanId() {
 			return Number(this.$route.params.id || 0);
 		},
+		enabledExperiment() {
+			const hasLentBefore = this.cookieStore.get(hasLentBeforeCookie);
+			const hasDepositBefore = this.cookieStore.get(hasDepositBeforeCookie);
+			return hasLentBefore !== 'true' && hasDepositBefore !== 'true' && !this.hasEverLoggedIn;
```

Comment:
Are these extra cookie checks necessary? Won't `hasLentBefore` and `hasDepositBefore` always be false if `!this.hasEverLoggedIn;`